package.json

```diff
@@ -18,7 +18,7 @@
     "react-router": "^6.22.1",
     "react-router-dom": "^6.22.1",
     "react-scripts": "5.0.1",
-    "reactflow": "^11.10.4",
+    "reactflow": "^11.11.1",
```

Comment:
Please do not upgrade package unless you intended to

Reply:
The old version does not run on my computer

Reply:
try to `rm -rf node_modules` and then reinstall

Reply:
fixed